static/js/validators.js

```diff
@@ -0,0 +1,14 @@
+const errorMessage = document.querySelector('#contact-form-errors');
+function showError(error) {
+  errorMessage.innerHTML = `<p>${error}`;
+}
+export function removeErrorMessage() {
+  errorMessage.innerHTML = '';
+}
+export const validateEmail = (e) => {
```

Comment:
- [ ] Let's review the construction of our validation. As it is right now is impeding to submit e-mails even when they're correct.

![image](https://user-images.githubusercontent.com/79114233/141138786-e9ef8b74-2412-4b1b-8a5a-9d335e60dc35.png)